spring-cloud-starter-task-composedtaskrunner/src/main/java/org/springframework/cloud/task/app/composedtaskrunner/properties/ComposedTaskProperties.java

```diff
@@ -215,4 +222,12 @@ public boolean isSplitThreadWaitForTasksToCompleteOnShutdown() {
 	public void setSplitThreadWaitForTasksToCompleteOnShutdown(boolean splitThreadWaitForTasksToCompleteOnShutdown) {
 		this.splitThreadWaitForTasksToCompleteOnShutdown = splitThreadWaitForTasksToCompleteOnShutdown;
 	}
+
+	public boolean isUniqueInstanceEnabled() {
+		return uniqueInstanceEnabled;
+	}
+
+	public void setUniqueInstanceEnabled(boolean uniqueInstanceEnabled) {
```

Comment:
See previous about the name of this field.

Reply:
fixed